lib/schemaUtils.js

```diff
@@ -137,9 +137,16 @@ function safeSchemaFaker(oldSchema, resolveTo, resolveFor, parameterSourceOption
   var prop, key, resolvedSchema, fakedSchema,
     schemaResolutionCache = _.get(schemaCache, 'schemaResolutionCache', {}),
     schemaFakerCache = _.get(schemaCache, 'schemaFakerCache', {});
+  let concreteUtils = DEFAULT_SCHEMA_UTILS;
```

Comment:
if we define the concrete utils in schemapack why do we need to check here again?